docs/manual/troubleshooting.rst

```diff
@@ -71,7 +71,72 @@ Qtile, so that the debug logs printed by Qtile are only the server's.
 
 If you suspect a client may be responsible for a bug, it can be helpful to look
 at the issue trackers for other compositors, such as `sway
-<https://github.com/swaywm/sway/issues`_. Similarly if you're hacking on
+<https://github.com/swaywm/sway/issues>`_. Similarly if you're hacking on
 Qtile's internals and think you've found an unexpected quirk it may be helpful
 to search the issue tracker for `wlroots
 <https://gitlab.freedesktop.org/wlroots/wlroots/-/issues>`_.
+
+Common Issues
+=============
+
+Cairo errors
```

Comment:
Should this link to the faq or vice versa? Don't think we need it twice.

Reply:
Based on how many people have had a problem with this I'd rather have it twice :grinning:. But yeah I'll link the FAQ to here.

Reply:
I agree that it's very important for it to be in the docs. Only issue is that it's slightly harder to maintain if the text is written in multiple places.